.eslintrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "extends": [
+    "eslint:recommended",
```

Comment:
next recommends putting this first

https://nextjs.org/docs/app/building-your-application/configuring/eslint#additional-configurations